sympy/matrices/tests/test_matrices.py

```diff
@@ -128,6 +128,15 @@ def test_multiplication():
     assert c[2, 0] == 18
     assert c[2, 1] == 0
 
+    if PY3:
```

Comment:
This is invalid syntax in < 3.5. You will have to test the operator itself using `eval`.